CHANGELOG.md

```diff
@@ -62,7 +62,7 @@
 - Added support for named aggregations in `DataFrame.aggregate` and `Series.aggregate` with `axis=0`.
 - `pd.read_csv` reads using the native pandas CSV parser, then uploads data to snowflake using parquet. This enables most of the parameters supported by `read_csv` including date parsing and numeric conversions. Uploading via parquet is roughly twice as fast as uploading via CSV.
 - Initial work to support an Index directly in Snowpark pandas. Support for Index as a first-class component of Snowpark pandas is coming soon.
```

Comment:
Why is Index capitalized?

Reply:
@sfc-gh-nkrishna would you have more context here? would adding `pd.Index` make more sense here?